docs/general/clients/jellyfin-vue.md

```diff
@@ -0,0 +1,185 @@
+Learning about [Compose](https://docs.docker.com/compose) is also recommended.
+* Docker Compose is now shipped with Docker, so you don't need to install it. It's recommended that you uninstall
+the old `docker-compose`.
+* [`version` key is deprecated](https://docs.docker.com/reference/compose-file/version-and-name/#version-top-level-element-obsolete)
+in Docker Compose, hence not included below.
+
+We're going to use the following `docker-compose.yml` as a starting point:
+
+```yaml
+services:
+  jellyfin_vue:
+    container_name: jellyfin_vue
+    image: ghcr.io/jellyfin/jellyfin-vue:unstable
+    restart: always
+    ports:
+      - 8080:80
+    labels:
+      - "com.centurylinklabs.watchtower.enable=true"
+
+  watchtower:
+    container_name: watchtower
+    image: ghcr.io/containrrr/watchtower
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    environment:
+      TZ: Europe/Madrid
+      WATCHTOWER_CLEANUP: 1
+      WATCHTOWER_INCLUDE_RESTARTING: 1
+      WATCHTOWER_POLL_INTERVAL: 60
+      WATCHTOWER_LABEL_ENABLE: 1
+    # Needed so watchtower doesn't restart-loop when updating itself
+    restart: on-failure
+```
+
+<span id="why-updates-needed"></span>
+
+:::success
+Getting things up and running is as easy as doing `docker compose up` with your terminal
+located where the `docker-compose.yml` file is.
+:::
+
+With this compose file:
+
+* Jellyfin Vue will be accessible on `8080` port of the machine that's running the containers
+* Watchtower takes care of updating the container to the latest commit available on the [repository](https://github.com/jellyfin/jellyfin-vue).
+This is a good idea because:
+  * As of now, stable releases don't exist and there have only been prereleases that don't necessarily meet a quality criteria, but
+    [major development milestones](https://jellyfin.org/posts/vue-vue3/).
+  * Only the latest `unstable` image is supported.
+
+After accessing the instance in your browser of choice, you'll be prompted to add a server. You can use your own server or our demo
+instance, located at `https://demo.jellyfin.org/stable`
+
+:::note
+The server address you need to type is relative to the device you're accessing Jellyfin Vue. For instance,
+if your Jellyfin Server is located at 192.168.0.10 and you have deployed Jellyfin Vue to that same server,
+and your client is 192.168.0.20, the address you need to input is `http://192.168.0.10`, not
+`http://127.0.0.1` or `http://localhost`.
+
+You can learn more about how the connection to your server works
+[in Jellyfin's Vue repository privacy disclaimer](https://github.com/jellyfin/jellyfin-vue#privacy-disclaimer-)
+:::
+
+### Using environment variables ###
+
+This is an example of how environment variables are set in the container by adding the
+following keys to the `jellyfin_vue` definition in [above's](#recommended-using-docker) `docker-compose.yml`:
```

Comment:
```suggestion
following keys to the `jellyfin_vue` definition in [the above `docker-compose.yml`](#recommended-using-docker):
```
verify that monospace renders properly inside links tho
I'm suggesting moving the file name into the link for accessibility reasons (never use «for more info [click here]» style links)

Reply:
Really good idea and they're rendered properly!